### ISSUE TITLE
[BUG] - Crop options doesn't show up.

### ISSUE THREAD

**Issue by Chandra-Sekhar-Bala** [CanHub/Android-Image-Cropper#428]
🚨🚨 Before opening the ticket please check if you can reproduce the bug in the sample code

 - Lib Version [4.3.1]
 
**Describe the bug**
Crop options does not show up if theme changed to ```NoActionBar```

**To Reproduce**

1. Change app theme to NoActionBar
2. Launch CropImageContract();
3. Select any image
4. No opinion will b shown

**Expected behavior**
options should be shown, as usual.

**Media**
Please add screenshots and/or screen recordings to help explain your problem.
We cannot help without more information
> Working fine
<img width="446" alt="WORKED" src="https://user-images.githubusercontent.com/57448981/186261675-3021b13a-4b92-4f12-989b-882c2b7cd1f5.png">

> Doesn't work
<img width="405" alt="NOT_WORKED" src="https://user-images.githubusercontent.com/57448981/186261761-299832aa-ef54-41a6-bec3-8e505600d608.png">

**Smartphone (please complete the following information):**
 - Device: [ Pixel 4 ]
 - OS: [Android 11]

**Additional context**
I tried to click all those top blank areas, it didn't worked.

**Comment by Canato** [comment]
Hey @Chandra-Sekhar-Bala check if this [FAQ](https://github.com/CanHub/Android-Image-Cropper/blob/main/.documentation/FAQ.md#cant-see-the-crop-button-so-cant-crop-the-image) can help you and let me know

I think is the first question.

**Comment by Chandra-Sekhar-Bala** [comment]
Thanks @Canato  🙌🏽

**Comment by tarun-patch** [comment]
the FAQ page doesn't exist. the URL is giving 404. I am also facing the same issue. @Canato

**Comment by vanniktech** [comment]
You need to set an app theme with toolbars.

**Comment by hiteshsarsava** [comment]
@vanniktech Atleast Add some example to roll own activity. Also Add disclaimer in notes with Bold Letters. I wasted 2 hours for this. Because I am using No Action bar Activity Theme in App. Now these days all the devs using this practice.

**Comment by Canato** [comment]
Hey @hiteshsarsava thanks for posting the same message 3x, it really help us with the spamming messages we received when helping open source projects in a volunteer free time schedule

Show us what is really important.

I answered you here: https://github.com/CanHub/Android-Image-Cropper/issues/433#issuecomment-2392521626

I linked a sample and offer the idea for you to open a PR with improvements.

I hope in the future you don't get frustrated trying to understand something new if take more than 2hours.
